fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -389,7 +397,11 @@ def upload(app_id, dir)
         handle_error(@password)
       end
 
-      result
+      return result
+    end
+
+    def returned_errors
```

Comment:
You mentioned that you would like to get rid of `@errors_to_show`. Could we rename this to `displayable_errors` and have it do something like...
```rb
@transporter_executor.map({ |error| "[Transporter Error Output]: #{error}" }).join("\n").gsub!(/"/, "")
```

Reply:
Thanks Josh, I've made the changes for this. This gets useful messaging out to slack now, however I think the logging/messaging is a bit too verbose and we are essentially duping error logs because `UI.crash!("Return status of iTunes Transporter was #{$1}: #{@errors.join('\n')}")` is essentially dumping the same message into @errors that is already there